demo-circuit/src/naive_threshold_sig_w_key_rotation/data_structures.rs

```diff
@@ -9,6 +9,9 @@ use primitives::{
 
 use crate::{common::NULL_CONST, Error};
 
+pub const VALIDATOR_HASH_SALT: u8 = 0u8;
```

Comment:
I'm not fully comfortable with this, as hardcoding it inside the circuit means that if and when we change the protocol, we need also to change the circuit and keep the backward compatibility with the previous one(s).
It's also true that if we change the protocol, most likely also the circuit changes so perhaps I'm overthinking it.